### ISSUE TITLE
 An error occurred while transforming the Human3.6M preprocessed dataset.

### ISSUE THREAD

**Issue by xiao19971225** [facebookresearch/VideoPose3D#122]
prepare_data_h36m.py:66: H5pyDeprecationWarning: The default file mode will change to 'r' (read-only) in h5py 3.0. To suppress this warning, pass the mode you need to h5py.File(), or set the global default h5.get_config().default_file_mode, or set the environment variable H5PY_DEFAULT_READONLY=1. Available modes are: 'r', 'r+', 'w', 'w-'/'x', 'a'. See the docs for details.
  with h5py.File(f) as hf:
prepare_data_h36m.py:67: H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
  positions = hf['3D_positions'].value.reshape(32, 3, -1).transpose(2, 0, 1)

Can you provide the following documents?
data_2d_h36m_gt.npz     data_2d_h36m_sh_ft_h36m.npz    data_2d_h36m_sh_pt_mpii.npz  
data_3d_h36m.npz

**Comment by xiao19971225** [comment]
Then it is not a video.
>python run.py -k cpn_ft_h36m_dbb -arc 3,3,3,3,3 -c checkpoint --evaluate pretrained_h36m_cpn.bin
Namespace(actions='*', architecture='3,3,3,3,3', batch_size=1024, bone_length_term=True, by_subject=False, causal=False, channels=1024, checkpoint='checkpoint', checkpoint_frequency=10, data_augmentation=True, dataset='h36m', dense=False, disable_optimizations=False, downsample=1, dropout=0.25, epochs=60, evaluate='pretrained_h36m_cpn.bin', export_training_curves=False, keypoints='cpn_ft_h36m_dbb', learning_rate=0.001, linear_projection=False, lr_decay=0.95, no_eval=False, no_proj=False, render=False, resume='', stride=1, subjects_test='S9,S11', subjects_train='S1,S5,S6,S7,S8', subjects_unlabeled='', subset=1, test_time_augmentation=True, viz_action=None, viz_bitrate=3000, viz_camera=0, viz_downsample=1, viz_export=None, viz_limit=-1, viz_no_ground_truth=False, viz_output=None, viz_size=5, viz_skip=0, viz_subject=None, viz_video=None, warmup=1)
Loading dataset...
Preparing data...
Loading 2D detections...
INFO: Receptive field: 243 frames
INFO: Trainable parameter count: 16952371
Loading checkpoint checkpoint\pretrained_h36m_cpn.bin
This model was trained for 80 epochs
INFO: Testing on 543344 frames
Evaluating...
----Directions----
Test time augmentation: True
Protocol #1 Error (MPJPE): 45.156178012348185 mm
Protocol #2 Error (P-MPJPE): 34.119459297758915 mm
Protocol #3 Error (N-MPJPE): 42.629377425350704 mm
Velocity Error (MPJVE): 2.9753523432121494 mm
----------
----Discussion----
Test time augmentation: True
Protocol #1 Error (MPJPE): 46.67484980633454 mm
Protocol #2 Error (P-MPJPE): 36.12383959144466 mm
Protocol #3 Error (N-MPJPE): 45.07634816695338 mm
Velocity Error (MPJVE): 3.1058947768727285 mm
----------

**Comment by lunalulu** [comment]
hi , https://www.dropbox.com/s/e35qv3n6zlkouki/h36m.zip is 404 now. Can you provide me the preprocessed data? email is luckygirl_777@126.com,thanks very much~

**Comment by sumitsarkar1** [comment]
I also need the **data_3d_h36m.npz** ..officially [website](https://vision.imar.ro/human3.6m/checklogin.php) says "account needs manual verification by admins"...my email id mit4allofu@gmail.com

**Comment by xiao19971225** [comment]
1143535596@qq.com
 
From: Sumit Sarkar
Date: 2020-11-24 10:55
To: facebookresearch/VideoPose3D
CC: xiaosubei; State change
Subject: Re: [facebookresearch/VideoPose3D] An error occurred while transforming the Human3.6M preprocessed dataset. (#122)
I also need the data_3d_h36m.npz ..officially website says "account needs manual verification by admins"
—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub, or unsubscribe.


-----------------------------------------------------------
以下文件通过foxmail发给您。保存时间有限制，请及时提取。
data_3d_h36m.npz (174.2 M) (有效时间到: 2020/12/10 22:31:23)
链接: http://mail.qq.com/cgi-bin/ftnExs_download?k=26396463af6b78c78dce4c271731571d565d01070502560a4c0e545b051c5156070149015302061f025f065b54080657515b525431216556004d053c02553a5a520f094d5f411f322b4d33&t=exs_ftn_download&code=a9dc11e2

**Comment by shineway14** [comment]
would you please send me one,thanks
408780518@qq.com

**Comment by xiao19971225** [comment]
> 请给我寄一份，谢谢
> [408780518@qq.com](mailto:408780518@qq.com)

which one?

**Comment by shineway14** [comment]
> > 请给我寄一份，谢谢
> > [408780518@qq.com](mailto:408780518@qq.com)
> 
> which one?

data_3d_h36m.npz (174.2 M)

**Comment by RCpengnan** [comment]
would you please send me one,thanks 2450856237@qq.com

**Comment by xiao19971225** [comment]
***@***.***
 
From: RCpengnan
Date: 2021-08-25 21:02
To: facebookresearch/VideoPose3D
CC: xiaosubei; State change
Subject: Re: [facebookresearch/VideoPose3D] An error occurred while transforming the Human3.6M preprocessed dataset. (#122)
would you please send me one,thanks ***@***.***
—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.


-----------------------------------------------------------
以下文件通过foxmail发给您。保存时间有限制，请及时提取。
data_3d_h36m.npz (174.2 M) (有效时间到: 2021/9/19 19:03:42)
链接: http://mail.qq.com/cgi-bin/ftnExs_download?k=7f353737fd6c2bc0d4c21f734536041a08000e5502065257150555015b1b025709541a0f05010f18090754065a05060059540e5363263651594156685052695d0b035a190d464c35724160&t=exs_ftn_download&code=8577c665

**Comment by RCpengnan** [comment]
Thank you for your data.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "facebookresearch/VideoPose3D"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月3日(星期五) 晚上7:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [facebookresearch/VideoPose3D]  An error occurred while transforming the Human3.6M preprocessed dataset. (#122)





 
 
 
 
 ***@***.*** 
   
 From: RCpengnan 
 Date: 2021-08-25 21:02 
 To: facebookresearch/VideoPose3D 
 CC: xiaosubei; State change 
 Subject: Re: [facebookresearch/VideoPose3D] An error occurred while transforming the Human3.6M preprocessed dataset. (#122) 
 would you please send me one,thanks ***@***.*** 
 — 
 You are receiving this because you modified the open/close state. 
 Reply to this email directly, view it on GitHub, or unsubscribe. 
 Triage notifications on the go with GitHub Mobile for iOS or Android. 
 
 
 ----------------------------------------------------------- 
 以下文件通过foxmail发给您。保存时间有限制，请及时提取。 
 data_3d_h36m.npz (174.2 M) (有效时间到: 2021/9/19 19:03:42) 
 链接: http://mail.qq.com/cgi-bin/ftnExs_download?k=7f353737fd6c2bc0d4c21f734536041a08000e5502065257150555015b1b025709541a0f05010f18090754065a05060059540e5363263651594156685052695d0b035a190d464c35724160&amp;t=exs_ftn_download&amp;code=8577c665 
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by muhabdulhaq** [comment]
If anyone have the the data_2d_h36m_gt.npz and data_3d_h36m.npz file, can you please share to me,
haq-muhammad@ed.tmu.ac.jp

Thank you,

**Comment by mmahdavian** [comment]
Hi. I need data_2d_h36m_gt.npz and data_3d_h36m.npz file. I appreciate it if anyone could send me them.
mmahdavi@sfu.ca

**Comment by wenwen12321** [comment]
Hi, I need data using stacked hourglass detections.

But I can't run following script which provided by author 
```
python prepare_data_2d_h36m_sh.py -pt h36m.zip
 python prepare_data_2d_h36m_sh.py -ft stacked_hourglass_fined_tuned_240.tar.gz
```

because I don't have both "**h36m.zip**" and "**stacked_hourglass_fined_tuned_240.tar.gz**" files.
I appreciate it if anyone could send me them.
